src/windows/loadsave.c

```diff
@@ -389,43 +394,71 @@ static void window_loadsave_scrollmouseover(rct_window *w, int scrollIndex, int
 
 static void window_loadsave_textinput(rct_window *w, int widgetIndex, char *text)
 {
+	if (text == NULL || text[0] == 0)
```

Comment:
Could you move this under the declaration again? It's convention to have these either locally or at the top of the function.